src/mirakuru/base.py

```diff
@@ -349,8 +349,15 @@ def check_timeout(self):
 
     def __del__(self):
         """Cleanup subprocesses created during Executor lifetime."""
-        if self.process:
-            self.kill()
+        try:
+            if self.process:
+                self.kill()
+        except Exception:
+            print("*" * 80)
+            print("Exception while deleting Executor.  It is strongly suggested that you use")
```

Comment:
`Executor.  It` There are two spaces in between.
Plus the full line is too long. 

Consider this:
```
print(
    'Exception while deleting Executor. '
    'It is strongly suggested that you use'
)